doc/Extensions/RRDCached.md

```diff
@@ -124,6 +124,46 @@ OPTS="$OPTS -w 1800 -z 1800 -f 3600 -t 4"
 ```php
 $config['rrdcached'] = "unix:/var/run/rrdcached.sock";
 ```
+### RRDCached installation Debian Stretch (rrdcached 1.6.0)
+
+1. Install rrdcached
+```bash
+sudo apt-get install rrdcached
+```
+
+2. Edit /etc/default/rrdcached to include:
+```bash
+DAEMON=/usr/bin/rrdcached
+WRITE_TIMEOUT=1800
+WRITE_JITTER=1800
+WRITE_THREADS=4
+BASE_PATH=/mnt/librenms/rrd/db
+JOURNAL_PATH=/mnt/librenms/rrd/journal/
+PIDFILE=/var/run/rrdcached.pid
+SOCKFILE=/var/run/rrdcached.sock
+SOCKGROUP=librenms
+DAEMON_GROUP=librenms
+DAEMON_USER=librenms
+BASE_OPTIONS="-B -F -R"
+```
+
+3. Restart the rrdcached service
+```bash
+    systemctl restart rrdcached.service
+```
+
+4. Edit /opt/librenms/config.php to include:
+
+For local RRDCached server
+```php
+$config['rrdtool_version'] = '1.5.5';
```

Comment:
This should be set to the version installed, so 1.6.0 in this case (don't worry about the minor version)

Reply:
well i banged my head on that last 2 days but when it is set to 1.6.0 not a single graph on the overview tab is drawn. so that might be a bug if it supose to be working

Reply:
Possibly, we'd need more info for that but that doesn't mean we should put the wrong info in docs.

Reply:
totaly agree but my thoughts on the version variable where if above 1.5.5 put that as variable. after i managed to get it working that way :) 

but the error i got when having 1.6.0 in the config was. after i did revert it to 1.5.5 in the config it worked.
RRDTool Output ERROR: can not parse an empty rpn expression

Reply:
But then the config is not what version is installed and if we add any further checks later on this could break things - which if it was polling then people lose data.

If this surfaces someone with this setup who can help debug the issue then we can do that rather than purposefully tell people to set the wrong config from day one.

Reply:
did remove the lines ill try to find the root cause tonight. would be great to have someone troubleshoot this with me